src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -210,7 +210,7 @@ def split_sai_id_key(sai_id_key):
 
 def config(**kwargs):
```

Comment:
>config [](http://example.com/codeflow?start=4&length=6)

this process is general service, which support both TCP port and unix socket. and the choice is on the command line.

We can make preference of `unix_socket_path` above `host`/`port`, if all 3 are provided. No need to add `use_unix_socket_path` into service command line.

Reply:
Fixed - I hope it is ok